docs/deprecated.md

```diff
@@ -558,15 +558,34 @@ for the old option will be removed in a future release.
 
 **Deprecated in Release: v19.03**
 
-**Target For Removal In Release: v20.10**
+**Disabled by default in Release: v26.0**
+
+**Target For Removal In Release: v27.0**
 
 The image manifest
 [v2 schema 1](https://github.com/docker/distribution/blob/fda42e5ef908bdba722d435ff1f330d40dfcd56c/docs/spec/manifest-v2-1.md)
-format is deprecated in favor of the
-[v2 schema 2](https://github.com/docker/distribution/blob/fda42e5ef908bdba722d435ff1f330d40dfcd56c/docs/spec/manifest-v2-2.md) format.
-
-If the registry you are using still supports v2 schema 1, urge their administrators to move to v2 schema 2.
+and "Docker Image v1" formats were deprecated in favor of the
+[v2 schema 2](https://github.com/docker/distribution/blob/fda42e5ef908bdba722d435ff1f330d40dfcd56c/docs/spec/manifest-v2-2.md) and [OCI image spec](https://github.com/opencontainers/image-spec/tree/v1.1.0) formats.
```

Comment:
Let's link to the published spec at https://distribution.github.io/distribution/spec/manifest-v2-2/

Reply:
Ah, good one; I updated both links to their website